src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -76,9 +76,16 @@
     <Compile Include="$(CommonPath)\System\IO\PathInternal.CaseSensitivity.cs">
       <Link>Common\System\IO\PathInternal.CaseSensitivity.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs">
+      <Link>Common\System\Threading\Tasks\TaskToApm.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <!-- .NET Standard 1.7 -->
+  <ItemGroup Condition="'$(PackageTargetFramework)' == 'netstandard1.7' and '$(TargetGroup)' != 'net46' and '$(TargetGroup)' != 'net463'" >
```

Comment:
This will change once Eric's changes for harvesting stable packages are introduced. I do agree with Wes that for the time being the best condition for this would be `'$(TargetGroup)'==''`